mq/plugins/auditdFixup.py

```diff
@@ -101,6 +101,13 @@ def onMessage(self, message, metadata):
                 message['details']['gidstring'] = message['details']['gid']
                 del message['details']['gid']
 
+       # fix details.dhost to be hostname
+       if 'details' in message.key() and isinstance(message['details'], dict):
```

Comment:
This line has a syntax error, there is no method called "key()"
What you want instead, is to do something like this:
```
if 'details' in message:
```

The same goes for when you verify if 'dhost' is in message['details']

Reply:
it's a typo, should have been message.keys() which is in other places in this script, with that said, does that mean that those other fixups that contain message.keys() are not working? I've used message.keys() in other places and it does work in that regard.

Reply:
I just checked other events where message.keys() is used to transform a field, and the other locations that this is present does appear to be functional, so I think just correcting the typo is all that's needed here.

Reply:
This if statement works just fine:
```
if 'details' in message.keys():
```
The proposed syntax works the same as above, just seems to be more pythonic. With that being said, the typo fix will be sufficient